crates/oxc_linter/src/rules/jest/no_focused_tests.rs

```diff
@@ -137,12 +166,47 @@ fn test() {
         ("fit.each`table`()", None),
     ];
 
-    let fix = vec![
+    let mut fix = vec![
         ("describe.only('foo', () => {})", "describe('foo', () => {})", None),
         ("describe['only']('foo', () => {})", "describe('foo', () => {})", None),
         ("fdescribe('foo', () => {})", "describe('foo', () => {})", None),
     ];
 
+    let pass_vitest = vec![
+        (r#"it("test", () => {});"#, None),
+        (r#"describe("test group", () => {});"#, None),
+        (r#"it("test", () => {});"#, None),
+        (r#"describe("test group", () => {});"#, None),
+    ];
+
+    let fail_vitest = vec![
+        (
+            r#"
+            import { it } from 'vitest'; 
```

Comment:
Import from `vitest` or config file contains rules starts with 'vitest', we will treat it as vitest and modify the plugin name.